documentation/reference/consent_decision_making.md

```diff
@@ -0,0 +1,570 @@
+process. The differences come from two directions:
+
+- The needs of a largely asynchronously coordinated base of contributors
+- Internal feedback and iteration on the initial process, even after
+  modification for asynchrony
+
+> ## When to follow this process
+>
+> Whether this process is followed for any given proposal or discussion is left
+> entirely to the discretion of the people relevant to the discussion. We
+> believe that this process helps the project's contributors make expedient and
+> clear decisions. However, it should not become an obstacle that is followed
+> for every tiny decision. If a decision can efficiently be made without this
+> process, contributors are invited to do so.
+>
+> For the purposes of project planning, however, this process should be the
+> default and should be the strong preference, keeping in mind that it can be
+> made faster via [round shortcutting](#shortcutting-a-round) and the
+> [synchronous lightening process](#optional-synchronous-lightening-process).
+
+## Continuous improvement and exceptions to every rule
+
+**This process is not set in stone.** Nor is it a set of laws that must be
+followed at all times. This document uses prescriptive language filled with
+imperatives primarily to prevent the process itself from being filled with
+contingencies at every turn. However, as with every other aspect of our ever
+evolving processes, these are merely suggestions that we expect contributors to
+adhere to. They cannot cover every situation and undoubtedly there are
+exceptional situations where following these suggestions to the letter would be
+antithetical to the goals of the project and even of the process itself.
+
+If during your time as an Openverse contributor, you feel there are ways in
+which this process can be improved and iterated upon, please know that this
+process should be iterated upon and that suggestions for improvement are open
+from any contributor to the project.
+
+## Terms
+
+- "Author": The person making the proposal.
+- "Participants": All participants of the decision-making process, including the
+  author, the requested participants, and ad-hoc participants.
+- "Reactions": A broad class of feedback that can include personal preferences
+  as well as general doubts about the validity or safety of the solutions
+  described in the proposal.
+- "Question": A request for clarification or explanation of an aspect of the
+  proposal. Questions may also request additional information be added to the
+  proposal.
+- "Objection": An explained uncertainty about the appropriateness, efficacy, or
+  safety of a solution
+- "Paramount objection": An objection outlining an issue that would cause harm
+  to the project or its contributors; especially issues that could cause harm
+  and are difficult to reverse and not strictly necessary to accomplish the
+  goals of the project
+- "Ratified proposal": A proposal that has undergone the formal decision-making
```

Comment:
Would "Approved" work instead of "Ratified" document-wide? If so it might be nice since it overlaps with language already familiar to GitHub users (PR approval, for example).

Reply:
Sure.